Makefile

```diff
@@ -40,6 +42,12 @@ dev-image: build
 image:
 	docker build -t $(IMAGE) .
 
+push-container: image
```

Comment:
can you make it explicit like push-container-to-gcloud or something along these lines so its explicit?

Reply:
Changes have been made and tested.